test/TestApps/PushNotificationsDemoApp/main.cpp

```diff
@@ -10,7 +10,15 @@
 #include <propkey.h> //PKEY properties
 #include <propsys.h>
 #include <ShObjIdl_core.h>
+#include "WindowsAppRuntime.Test.AppModel.h"
 #include <WindowsAppRuntime.Test.Bootstrap.h>
+#include <WindowsAppRuntime.SelfContained.h>
```

Comment:
Why are these includes in main.cpp? I don't see any code added to main.cpp that references these newer includes

Reply:
AppModel can be removed, but self contained is supposed to check if the env is self-contained. I am going to add a check here.

Reply:
Right. Lets not add to the header bloat if at all possible.